vocode/streaming/models/transcriber.py

```diff
@@ -100,3 +101,6 @@ class AssemblyAITranscriberConfig(
     TranscriberConfig, type=TranscriberType.ASSEMBLY_AI.value
 ):
     pass
+
+class RevAITranscriberConfig(TranscriberConfig, type=TranscriberType.REV_AI):
+    downsampling: Optional[int] = None
```

Comment:
```suggestion
    pass
```

no need to override this

Reply:
@ajar98 Done!